charts/k8ssandra/dashboards/stargate.json

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
I'm not sure how or why this commit got included on this PR; I committed it on a different branch as part of #282.